src/functions-and-arrays.js

```diff
@@ -47,6 +134,17 @@ const wordsCount = [
   'matter'
 ];
 
+function howManyTimes(array, word){
+  let count =0
+  for(let i=0; i<array.length; i++){
+if (array[i]===word){
+  count++
+}
+
+  }
+  return count
+}
+
```

Comment:
Nicely done Sophie!

An alternative solution is 
`function howManyTimes(arr, word) {

  let count = 0;
  for (let el of arr) {
    if (el === word) count++;
  }
  return count;
}`